src/v1/products.ts

```diff
@@ -255,9 +261,15 @@ export class Products extends ThBaseHandler {
   async select (options: SelectProductsQuery): Promise<ProductsResponse> {
     try {
       const base = this.uriHelper.generateBaseUri('/selection')
-      const uri = this.uriHelper.generateUriWithQuery(base)
 
-      const response = await this.http.getClient().post(uri, options)
+      // Note that this POST endpoint accepts a body containing product_ids
+      // and/or custom_ids and filters as part of the query string.
+      const query = options.query
```

Comment:
You can replace these 3 lines with just this one:
```
const { query, ...body } = options
```

Reply:
Ah yeah, nice one.